src/app/interfaces/access-control-list.ts

```diff
@@ -0,0 +1,38 @@
+import { Permissions } from './permissions';
+import { AclFlags } from './acl-flags';
+
+  
+  interface NFS41Flags {
```

Comment:
Can you please reformat this file (remove leading spaces, multiple empty lines)?